Classes/MDMPersistenceController/MDMPersistenceController.h

```diff
@@ -67,6 +67,24 @@ extern NSString *const MDMIndpendentManagedObjectContextDidSaveNotification;
 - (id)initWithStoreURL:(NSURL *)storeURL modelURL:(NSURL *)modelURL;
 
 /**
+ Returns an inMemory backed persistence controller initialized with the given arguments.
+ 
+ @param model A managed object model.
+ 
+ @return A new persistence controller object with an inMemoryStore type with model.
+ */
+- (id)initInMemoryTypeWithModel:(NSManagedObjectModel *)model;
```

Comment:
Should all the `init` methods, not just these new one, return an `instancetype` instead?